### ISSUE TITLE
Treat bundled database as any other `multi_epidist` object

### ISSUE THREAD

**Issue by Bisaloo** [epiverse-trace/epiparameter#199]
One of the reasons I find it difficult to wrap my head around epiparameter is because it has two scopes:

- provide a database of epi parameters (or link to it in the future)
- provide tools to work with probability distributions

These two scopes are sometimes entangled. `list_parameters()` is a good example for this. It can list parameters from the bundled database, or from another `multi_epidist` object.

I propose to clearly separate the two scopes by:

- having a single function to pull the database (e.g., `epidist_db()`)
- treating the data created by `epidist_db()` as any other `multi_epidist` object and not pull it under the hood for users.

In other words and from a practical point of view, it would no longer be possible to do:

```r
list_parameter(disease = "Ebola")
```

But you would now have to do:

```r
db <- epidist_db()
list_parameter(multi_epidist = db, disease = "Ebola")
```

There could also then be a case to move `multi_epidist` as the first argument to allow piping:

```r
epidist_db() |>
  list_parameter(disease = "Ebola")
```

The db could alternatively be provided as an exported dataset (loosely related to #198)

**Comment by joshwlambert** [comment]
Thanks for the suggestion. I'm happy to refactor the scope of the `list_distributions()` function as outlined in your comment. 

Is it also worth renaming the function to `list_parameter()` or `list_parameters()` (this is what you called it in the issue and perhaps it is a more intuitive naming). Alternatively, I could give `list_distributions()` an alias.
